src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,18 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def cve?
+    kind == "cve"
+  end
+
+  def verify_issue_name!(name)
```

Comment:
Nice! I love it, just one more thing.... don't kill me please :/

What about having the raise of the exception at patchinfo model instead at the method that validates? I mean, a method that validates only need to return a boolean. And then the patchinfo, that already raises another exception , will raise the proper exception.